tanip_pipeline.conf

```diff
@@ -10,27 +10,26 @@
     Path    ./app/logfile.log
```

Comment:
Should this path match up with what is being produced by the generator app?

Reply:
Same as above.